templates/resources.md.tmpl

```diff
@@ -62,7 +62,20 @@
 
 {{ if .SchemaMarkdown }}{{ .SchemaMarkdown | trimspace }}{{ end }}
 
-# Import 
-## Add the import
+## Import 
```

Comment:
so we're only updating resource doc template to include this in future resources?

From this [comment](https://jira.mongodb.org/browse/CLOUDP-273976?focusedCommentId=6733172&focusedId=6733172&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6733172) on the ticket I thought it was decided to have this in a 'Best practices' page? Or did that change?

Reply:
I wasn't sure for what `Best practices` page was meant. I was thinking about our internal development flow.
An alternative is adding directly to the "landing-page" of terraform registry: https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs
But this seemed a bit too obvious. Adding a point to our tech discussions

Reply:
sounds good, good to have an alignment on this intenrally